lib/svg/svg.py

```diff
@@ -14,20 +14,13 @@ def get_document(node):
     return node.getroottree().getroot()
 
 
-def generate_unique_id(document_or_element, prefix="path"):
+def generate_unique_id(document_or_element, prefix="path", blacklist=None):
```

Comment:
Linguistic nitpick: could we please change this to "blocklist"?

Reply:
Done